sdks/typescript/src/apache_beam/utils/service.ts

```diff
@@ -414,7 +414,7 @@ export class PythonService extends SubprocessService {
     const result = childProcess.spawnSync(
       PythonService.whichPython(),
       [bootstrapScript],
-      { encoding: "latin1" },
+      { encoding: "utf-8" },
```

Comment:
Is this safe?

Reply:
yes

Reply:
To clarify, how do we know the subprocess will always produce valid utf-8? (I chose latin1 because it has no "invalid" encodings.) If we can guarantee this, or at least it fails gracefully on invalid byte sequences, this is a definite improvement.

Reply:
TBH I made this change originally because I was getting encoding errors somewhere else in my tooling, it turned out not to be this, so I'll revert